install.sh

```diff
@@ -1,11 +1,24 @@
 #!/bin/sh
 set -e
 
-# Setup Python venv and install packages
-echo "Setting up Python virtual environment in ./.venv"
-python3.8 -m venv ./.venv
-. .venv/bin/activate
-python3.8 -m pip install -r environment/requirements.txt
+
+# Get the operating system name
+os_name=$(uname -o)
+
+print os_name
+# Check if the operating system is Windows
+if [[ "$os_name" == "Cygwin" || "$os_name" == "Msys" ]]; then
+    echo "Setting up Python virtual environment in ./.venv"
+    python -m venv ./.venv
+    . .venv/Scripts/activate
+    python -m pip install -r environment/requirements.txt
+else
+    # Setup Python venv and install packages
+    echo "Setting up Python virtual environment in ./.venv"
+    python3.8 -m venv ./.venv
+    . .venv/bin/activate
+    python3.8 -m pip install -r environment/requirements.txt
+fi
```

Comment:
```suggestion
echo "Setting up Python virtual environment in ./.venv"
python3.8 -m venv ./.venv

if [[ "$os_name" == "Cygwin" || "$os_name" == "Msys" ]]; then
    . .venv/Scripts/activate
else
    . .venv/bin/activate
fi

python -m pip install -r environment/requirements.txt
```

Let's keep the only windows-specific part in the `if` and everything else outside.

Reply:
when you use "python3.8 -m venve ./.venv" doesn't that by default requires the python version to be 3.8? using  "python3 -m venve ./.venv" or  "python -m venve ./.venv"  will resolve this problem

Reply:
@frankharkins install.sh is updated and submitted as requested

Reply:
@frankharkins  I made the changes requested, please review the latest changes

Reply:
Hi, sorry – yes we want to make sure all contributors are using the same version of Python. Don't worry though I can push this change now.